tasks/send-pan/src/index.ts

```diff
@@ -45,7 +45,12 @@ async function sendPAN(event: HandlerEvent): Promise<HandlerOutput> {
   const panName = input.pdr.name.replace(/\.pdr/gi, '.pan');
   const uploadPath = path.join(remoteDir, panName);
 
-  const pan = pdrHelpers.generatePAN();
+  if (input.running.length !== 0) {
+    throw new Error('Executions still running');
+  }
+
+  const executions = input.failed.map((exc) => exc.arn).concat(input.completed);
+  const pan = await pdrHelpers.generateShortPAN(executions);
```

Comment:
If the input payload already have the `running, failed, completed executions`, why do we need to pass execution arns to `pdrHelpers.generateShortPAN` and fetch them again?

Reply:
I was thinking ahead a bit for when long pan has to be implemented, it would be good to have all the error information available. But yeah I can switch to `if (input.failed.length) > 0` for short pan

Reply:
Also the failed input doesn't give a whole lot of information, So I fetched to get more
Example: 
```
    "running": [],
    "failed": [
        {
            "arn": "arn:aws:states:us-west-2:148260516034:execution:asdc2-sit-IngestUmmgGranule:c7315e0a-4f0b-45da-99ae-a6dd0e74aab6",
            "reason": "Workflow Failed"
        }
    ],
    "completed": [
        "arn:aws:states:us-west-2:148260516034:execution:asdc2-sit-IngestUmmgGranule:4bc1b0ce-29ef-4292-9d60-0a998a79974f"
    ],
```

Reply:
If the failure reason is needed later, we can update PdrStatusCheck task to detailed error message.
https://github.com/nasa/cumulus/blob/v18.5.1/tasks/pdr-status-check/index.js#L93

Reply:
I updated